geocoder-data-provider/caddy-minio-pvc-template.yaml

```diff
@@ -0,0 +1,382 @@
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          app: ${APPLICATION_NAME}
+          deploymentconfig: ${CADDY_SERVICE_NAME}
+      spec:
+        containers:
+        - args:
+          - /conf/Caddyfile
+          command:
+          - caddy
+          - -quic
+          - -conf
+          image: abiosoft/caddy:no-stats
+          imagePullPolicy: IfNotPresent
+          name: www
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+          - mountPath: /conf
+            name: caddy-conf
+          - mountPath: /data
+            name: app-volume
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        schedulerName: default-scheduler
+        securityContext: {}
+        terminationGracePeriodSeconds: 30
+        volumes:
+        - name: app-volume
+          persistentVolumeClaim:
+            claimName: ${APPLICATION_NAME}-pv
+            #claimName: minio-pv-claim
+        - configMap:
+            defaultMode: 420
+            items:
+            - key: caddifile
+              path: Caddyfile
+            name: caddy-conf-${APPLICATION_NAME}
+          name: caddy-conf
+    test: false
+    triggers:
+    - type: ConfigChange
+- apiVersion: apps.openshift.io/v1
+  #
+  # Minio DeploymentConfig
+  #
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 10
+    selector:
+      app: ${APPLICATION_NAME}
+      deploymentconfig: ${MINIO_SERVICE_NAME}
+    strategy:
+      activeDeadlineSeconds: 21600
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          app: ${APPLICATION_NAME}
+          deploymentconfig: ${MINIO_SERVICE_NAME}
+      spec:
+        containers:
+        - env:
+          - name: MINIO_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                key: minioAccessKey
+                name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+                #name: minio
+          - name: MINIO_SECRET_KEY
+            valueFrom:
+              secretKeyRef:
+                key: minioSecretKey
+                name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+                #name: minio
+          - name: MINIO_CONFIG_DIR
+            value: /tmp
+          - name: MINIO_DATA_DIR
+            value: /data
+          image: docker-registry.default.svc:5000/openshift/minio
```

Comment:
Also, looks like minio setup runs via `./minio server /data`, so we need to add the following to startup minio pod:
```
 args:
            - server
            - /data
```